bin/setup_remote

```diff
@@ -28,11 +28,14 @@ TARGET_REPO=$GIT_PEER
 TARGET_VERSION=$(cat etc/${TARGET^^}_VERSION)
 
 if [ -d $TARGET ]; then
-    echo Dir $TARGET/ already exists. Not cloning.
+    echo Dir $TARGET/ already exists. Fetching updates...
+    (cd $TARGET; git fetch)
 else
     echo Cloning $TARGET locally...
     git clone $TARGET_REPO $TARGET
 fi
 
 echo Switching to target version $TARGET_VERSION
```

Comment:
cosmetic, but s/target/$TARGET might make this easier to read in the logs.

Reply:
agreed